src/js/grapl-cdk/lib/services/dynamoDbTable.ts

```diff
@@ -0,0 +1,61 @@
+import * as cdk from '@aws-cdk/core';
+import * as dynamodb from '@aws-cdk/aws-dynamodb';
+import * as iam from '@aws-cdk/aws-iam';
+
+import { Service } from '../service';
+import { RemovalPolicy } from '@aws-cdk/core';
+import { FargateService } from "../fargate_service";
+
+export interface DbTableProps {
```

Comment:
This whole file should be in `lib/` not `lib/services`; services is strictly for, like, lambdas and such